src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write code here
+const treeItems = document.querySelectorAll('.tree li');
+
+treeItems.forEach((item) => {
+  const ul = item.querySelector('ul');
+
+  if (!ul) {
+    return;
+  }
+
+  const header = document.createElement('span');
+
+  header.textContent = item.firstChild.textContent.trim();
```

Comment:
Accessing `item.firstChild.textContent` may lead to unexpected results if the first child is not a text node. Consider using `item.firstChild?.textContent` to safely access the text content or check if `firstChild` is a text node before proceeding.